test/e2e/run_test.go

```diff
@@ -1138,7 +1138,7 @@ echo -n madeit-$teststring >$tmpfile
 	It("podman run exit code on failure to exec", func() {
 		session := podmanTest.Podman([]string{"run", ALPINE, "/etc"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(ExitWithError(126, "open executable: Operation not permitted: OCI permission denied"))
+		Expect(session).Should(ExitWithErrorRegex(126, ".*(open executable|the path `/etc` is not a regular file): Operation not permitted: OCI permission denied.*"))
```

Comment:
This here is the same exact test as `podman run exit ExecErrorCodeCannotInvoke`, given that we should drop on of them no point in testing it twice...

Reply:
dropped